src/applications/entities/application.entity.ts

```
@@ -2,10 +2,19 @@ import { User } from 'src/auth/entities/user.entity';
 import { JobStatus } from 'src/feed/enums/job-status.enum';
 import { Job } from 'src/jobs/entities/job.entity';
 import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, Unique, ManyToOne } from 'typeorm';
+import { BaseEntity } from 'typeorm'
```

ESLint

[error] Insert `;`

(prettier/prettier)